kernelci/api/helper.py

```diff
@@ -246,6 +259,46 @@ def submit_results(self, results, root):
         except requests.exceptions.HTTPError as error:
             raise RuntimeError(json.loads(error.response.text)) from error
 
+    def get_node_obj(self, node_dict, get_linked=False):
```

Comment:
If the objective here is to get an object from a dictionary, we can simply use `Node(**dict)` for it.
But I assume this would require importing model definitions and that wouldn't be possible due to python version conflicts.

Reply:
That's true.
It's good to add a comment to replace this function with `model.parse_node_obj` once we resolve python version issue.